pkgs/by-name/ry/rygel/package.nix

```diff
@@ -76,11 +81,13 @@ stdenv.mkDerivation (finalAttrs: {
       gtk3
       libmediaart
       pipewire
+      libX11
```

Comment:
Why is this needed when GUI is not built?

Reply:
An X11 function is explicitly called in the app unfortunately. See [this issue](https://gitlab.gnome.org/GNOME/rygel/-/issues/221). I've opened [a PR](https://gitlab.gnome.org/GNOME/rygel/-/merge_requests/27) that will hopefully allow removing the X11 dependency entirely.

Reply:
Could you please also add a comment linking to that next to the libx11 dependency?

Reply:
@Aleksanaa Want to do that in your PR?